app/models/data.py

```diff
@@ -139,8 +151,11 @@ class CoreData(db.Model, DataMixin):
     # composite PK: state_name, batch_id, date
     state = db.Column(db.String, db.ForeignKey('states.state'),
         nullable=False, primary_key=True)
+    state_obj = relationship("State", lazy="selectin")
```

Comment:
just curious if there's a reason for defining this differently than the way we have the batch relationship to CoreData? (ie we have a `coreData = relationship('CoreData', backref='batch')` on the Batch object)

Reply:
Not really I guess. It seemed like the "selectin" loading type offered some performance advantages in terms of not having to make separate queries, and it's compatible with sqlalchemy's cache that I don't think we're using, but I didn't profile it.